manager/logbroker/broker.go

```diff
@@ -260,13 +271,16 @@ func (lb *LogBroker) nodeConnected(nodeID string) {
 	lb.mu.Lock()
 	defer lb.mu.Unlock()
 
-	lb.connectedNodes[nodeID] = struct{}{}
+	lb.connectedNodes[nodeID] = make(map[*subscription]struct{})
```

Comment:
I'd first check whether the `connectedNodes` entry already exists, to avoid overwriting that map and orphaning existing subscriptions.